src/main/scala/com.snowplowanalytics/iglu.schemaddl/migrations/Migration.scala

```diff
@@ -97,14 +98,12 @@ object Migration {
     } yield cell
   }
 
-  def buildMigrationMatrix(schemas: NonEmptyList[IgluSchema]): NonEmptyList[OrderedSchemas] =
+  def buildMigrationMatrix(schemas: List[IgluSchema]): List[OrderedSchemas] =
```

Comment:
Would it be possible to make arguments NonEmptyList, rather than other way round?